components/TextFields/src/MDCTextInputControllerBase.m

```diff
@@ -397,13 +397,16 @@ - (void)updateLeadingUnderlineLabel {
 #pragma mark - TextInput Customization
 
 - (void)updateTextInput {
-  UIFont *font = self.textInputFont;
-  if (self.mdc_adjustsFontForContentSizeCategory) {
-    font = [font mdc_fontSizedForMaterialTextStyle:MDCFontTextStyleBody1
-                              scaledForDynamicType:_mdc_adjustsFontForContentSizeCategory];
-    // TODO: (#4331) This needs to be converted to the new text scheme.
-  }
-  self.textInput.font = font;
+	UIFont *font = self.textInputFont;
```

Comment:
Nit: Please correct the indentation for this method. It looks like it accidentally changed. It should be 2 spaces from the start of the line for method bodies.